### DISCUSSION TITLE
Commands not registered when building phar file

### DISCUSSION THREAD

**Discussion by underdpt** [lorisleiva/laravel-actions#243] [Q&A]
Hello,

I'm trying to use Laravel Actions on Laravel-Zero, but it seems it cannot scan the register the commands inside a Phar file.

This is my code to register the commands:

```php
        Actions::registerCommands(
            \Phar::running() !== ''
                ? \Phar::running(true) . '/app/Actions'
                : 'app/Actions'
        );
```

It works when testing the app, but once it's built it fails with this error:

```
In Finder.php line 659:
                                                                                 
  You must call one of in() or append() methods before iterating over a Finder.
```

The path when running the phar file is like:

```
phar://D:/project/builds/phar_file/app/Actions
```

So, It's there a way to register actions inside a phar file?

**Comment by edalzell** [comment]
Can you register your commands directly in the kernel?
